security/pfSense-pkg-suricata/files/usr/local/www/suricata/suricata_blocked.php

```diff
@@ -369,6 +395,9 @@
 				$rdns_link = "";
 				$rdns_link .= "<i class=\"fa fa-search icon-pointer\" onclick=\"javascript:resolve_with_ajax('{$block_ip_str}');\" title=\"";
 				$rdns_link .= gettext("Resolve host via reverse DNS lookup") . "\" alt=\"Icon Reverse Resolve with DNS\"></i>";
+				/* Add GeoIP check icon */
```

Comment:
Consider wrapping the display of the GeoIP check icon with a test to verify that GeoIP2 database download is enabled. The config variable is here:  $config['installedpackages']['suricata']['config'][0]['autogeoipupdate']. If that option is "on", display the GeoIP lookup icon, but if the option is "off", hide the GeoIP lookup icon.